cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -344,11 +383,12 @@ def data_preparation(
     google_trend_csv = spark.read.csv("%s/googletrend.csv" % data_dir_path, header=True)
     weather_csv = spark.read.csv("%s/weather.csv" % data_dir_path, header=True)
 
+    # a sampled subset of the train and test data is retrieved
     if hp.sample_rate:
         train_csv = train_csv.sample(withReplacement=False, fraction=hp.sample_rate)
         test_csv = test_csv.sample(withReplacement=False, fraction=hp.sample_rate)
 
-    # Prepare data frames from CSV files.
+    # prepare the DataFrames from the CSV files.
```

Comment:
Let's have the comment in either active/passive voice. This is in active voice, whereas the previous one is in passive voice.